### DISCUSSION TITLE
sysobject.ids, snmphostname and Toshiba Printer

### DISCUSSION THREAD

**Discussion by dsi66** [glpi-project/glpi-agent#564] [Ideas]
### Your idea

Hello

Can you add this oid for Toshiba printer?

1129.2.3.37.1 Toshiba PRINTER e-STUDIO2040C
1129.2.3.45.1 Toshiba PRINTER e-STUDIO2555C
1129.2.3.48.1 Toshiba PRINTER e-STUDIO287CS
1129.2.3.53.2 Toshiba PRINTER e-STUDIO257
1129.2.3.58.2 Toshiba PRINTER e-STUDIO2505AC
1129.2.3.68.2 Toshiba PRINTER e-STUDIO2515AC
1129.2.3.68.3 Toshiba PRINTER e-STUDIO3015AC
1129.2.3.69.4 Toshiba PRINTER e-STUDIO3518A
1129.2.3.69.5 Toshiba PRINTER e-STUDIO4518A
1129.2.3.76.2 Toshiba PRINTER e-STUDIO2525AC
1129.2.3.77.3 Toshiba PRINTER e-STUDIO3528A
1129.2.3.77.4 Toshiba PRINTER e-STUDIO4528A

For this printers, snmphostname is .1.3.6.1.4.1.1129.2.3.50.1.6.1.4.3.1.1.10.1.1

Thank you

**Comment by g-bougard** [comment]
Hi @dsi66 

thank you for your request.

Anyway is this still really required with the [latest GLPI Agent v1.7 we released yesterday](https://github.com/glpi-project/glpi-agent/releases/tag/1.7) ?
It includes an improvement for specifically Toshiba printers.
In the log, you may still see some partial match, but this is not an issue if the model is finally correct thanks to the new Toshiba module.
